test/simulations/NodeShortCircuit.jl

```diff
@@ -27,18 +26,16 @@ stat_lines = [
   StaticLine(;from=2, to=3, Y = 1/(0.1152 + im*0.0458)),
   StaticLine(;from=1, to=2, Y = 1/(0.1152 + im*0.0458))]
 
-nsc = NodeShortCircuit(R = 0.1, node = 1, sc_timespan=(1.,2.))
+nsc = NodeShortCircuit(node = 1, R = 0.1, sc_timespan=(1.,2.))
 
 pg_static = PowerGrid(busses, stat_lines)
 pg = PowerGrid(busses, lines)
 
-@test nsc(pg_static) == nothing
+#@test nsc(pg_static) == nothing
```

Comment:
Can you delete this line instead of uncommenting?

Reply:
Yes, I can do that. No problem.

Reply:
Thanks :)